nuxeo-java-client/src/main/java/org/nuxeo/client/NuxeoVersion.java

```diff
@@ -139,6 +142,13 @@ public NuxeoVersion hotfix(int hotfix) {
         return new NuxeoVersion(majorVersion, minorVersion, -1, hotfix, false);
     }
 
+    /**
+     * @return A new instance of {@link NuxeoVersion} with the same version + input minor number.
```

Comment:
```suggestion
     * @return a new instance of {@link NuxeoVersion} with a minor version increase.
```

Reply:
It is more a version override than an increase + it follows the previous method javadoc which has the same semantic.